### PR TITLE
1.0.0.Release

### PR DESCRIPTION
**Pull request by ractive**
Create a release version. This allows us to remove the snapshot repos from the backend project.

See https://github.com/local-ch/backend/pull/2405

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,11 +75,11 @@
...
 	<distributionManagement>
-		<snapshotRepository>
-			<id>nexus-snapshots</id>
-			<name>Internal Snapshots Releases</name>
-			<url>http://nexus.intra.local.ch:8081/nexus/content/repositories/snapshots/</url>
-		</snapshotRepository>
+		<repository>
+			<id>nexus</id>
+			<name>Internal Releases Repository</name>
+			<url>http://nexus.intra.local.ch:8081/nexus/content/repositories/releases//</url>
```
Comment (michaelkuechler) [inline pom.xml]: double slash in the end?!

**Review by michaelkuechler — COMMENTED** [review]
see comment

File: pom.xml (hunk 1 above)
Comment (ractive) [reply pom.xml]: image image

Comment (ractive) [reply pom.xml]: I removed one. But it actually also worked with both. ;-)

File: pom.xml [hunk 2]
```diff
@@ -23,8 +23,7 @@
 		<mockito.core.version>1.9.5</mockito.core.version>
 	</properties>
 
-	<dependencies>
-		<dependency>
+	<dependencies>		<dependency>
```
Comment (michaelkuechler) [inline pom.xml]: nice formatting…

**Pull request merged**